R/as_gt.R

```diff
@@ -280,17 +282,24 @@ as_gt.fixed_design <- function(x, title = NULL, footnote = NULL, ...) {
 #'     )
 #'   )
 #'
-#' # usage of display_bound = ...
+#' # Example 5 ----
+#' # Usage of display_bound = ...
 #' # to either show efficacy bound or futility bound, or both(default)
 #' gs_power_wlr() %>%
 #'   summary() %>%
 #'   as_gt(display_bound = "Efficacy")
 #'
-#' # usage of display_columns = ...
+#' # Example 6 ----
+#' # Usage of display_columns = ...
 #' # to select the columns to display in the summary table
 #' gs_power_wlr() %>%
 #'   summary() %>%
 #'   as_gt(display_columns = c("Analysis", "Bound", "Nominal p", "Z", "Probability"))
+#'
+#' # Example 7 ----
+#' # Customize the variables to be summarized for each analysis
+#' gs_power_wlr() %>%
```

Comment:
@LittleBeannie This is where the check error came from. This example is not complete.

Reply:
Thanks for the check! It is exactly where the error is. Thank you!